packages/cli/src/Server.ts

```diff
@@ -673,9 +673,9 @@ class App {
 			await WorkflowHelpers.validateWorkflow(updateData);
 			await Db.collections.Workflow!.update(id, updateData).catch(WorkflowHelpers.throwDuplicateEntryError);
 
-			const tablePrefix = config.get('database.tablePrefix');
-			await TagHelpers.removeRelations(req.params.id, tablePrefix);
-			if (tags?.length) {
+			if (tags) {
```

Comment:
If we remove `.length`, this will also be triggered when `tags` is `[]`, which is truthy. Same for L690.

Another option:
- receive empty array in `tags` → remove relations only
- receive 1+ item array in `tags` → remove and recreate relations

Reply:
sure, one less useless db query